app/src/main/java/com/duckduckgo/app/appearance/AppearanceViewModel.kt

```diff
@@ -63,6 +65,8 @@ class AppearanceViewModel @Inject constructor(
                 currentViewState().copy(
                     theme = themingDataStore.theme,
                     appIcon = settingsDataStore.appIcon,
+                    forceDarkModeEnabled = settingsDataStore.experimentalWebsiteDarkMode,
+                    canForceDarkMode = themingDataStore.theme != DuckDuckGoTheme.LIGHT,
```

Comment:
I think we should also check is theme is SYSTEM_DEFAULT, and system default turns out to be light